hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/util/FlinkClientUtil.java

```diff
@@ -56,6 +67,22 @@ public static org.apache.hadoop.conf.Configuration getHadoopConf() {
     if (hadoopConf == null) {
       hadoopConf = new org.apache.hadoop.conf.Configuration();
     }
+
+    String confDir = System.getenv(FLINK_CONF_DIR);
+    if (StringUtils.isNullOrEmpty(confDir)) {
+      confDir = System.getenv(ApplicationConstants.Environment.PWD.key());
```

Comment:
if FLINK_CONF_DIR not specified by user, we can just skip it. Loading conf from  `ApplicationConstants.Environment.PWD` by default does not look desirable - is this a common known behavior?

Reply:
Yes, it is by default how Flink Yarn load Flink configuration.

https://github.com/apache/flink/blob/master/flink-yarn/src/main/java/org/apache/flink/yarn/entrypoint/YarnApplicationClusterEntryPoint.java#L76

Reply:
We also have `HadoopConfigurations.getHadoopConf` to fetch hadoop conf during compile time, does it work as expected for your case?

Reply:
@danny0405 The solution is tested end to end internally to load CLAC configs in parquet. What do you mean "fetch hadoop conf during compile time"?

Reply:
@HuangZhenQiu So this is yarn-specific logic, we would need to add a comment here to clarify it.

> Yes, it is by default how Flink Yarn load Flink configuration.
> 
> https://github.com/apache/flink/blob/master/flink-yarn/src/main/java/org/apache/flink/yarn/entrypoint/YarnApplicationClusterEntryPoint.java#L76

Reply:
As discussed, see if `org.apache.hudi.configuration.HadoopConfigurations#getParquetConf` works for the scenario - normal `parquet.*` configs should be recognizable